docs/admin/scale/gke.md

```diff
@@ -0,0 +1,62 @@
+# Scaling Coder on Google Kubernetes Engine (GKE)
+
+This is a reference architecture for Coder on [Google Kubernetes Engine](#). We regurily load test these environments with a standard [kubernetes example](https://github.com/coder/coder/tree/main/examples/templates/kubernetes) template.
+
+> Performance and ideal node sizing depends on many factors, including workspace image and the [workspace sizes](https://github.com/coder/coder/issues/3519) you wish to give developers. Use Coder's [scale testing utility](./index.md#scale-testing-utility) to test your own deployment.
+
+## 50 users
+
+### Cluster configuration
+
+- **Autoscaling profile**: `optimize-utilization`
+
+- **Node pools**
+  - Default
+    - **Operating system**: `Ubuntu with containerd`
+    - **Instance type**: `e2-highcpu-8`
+    - **Min nodes**: `1`
+    - **Max nodes**: `4`
+
+### Coder settings
+
+- **Replica count**: `1`
+- **Provisioner daemons**: `30`
+- **Template**: [kubernetes example](https://github.com/coder/coder/tree/main/examples/templates/kubernetes)
+- **Coder server limits**:
+  - CPU: `2 cores`
+  - RAM: `4 GB`
+- **Coder server requests**:
+  - CPU: `2 cores`
+  - RAM: `4 GB`
+
+## 100 users
```

Comment:
Index.md claims that we tested at 50, 100, and 1000 users, but this doc only has 50, 100

Reply:
ah yea we should disregard the numbers and instance types at the moment. these are all placeholder-ish to align on format and info we want to display